### PR TITLE
Added guide to whitelist DB IP on AWS

### PR DESCRIPTION
**Pull request by AtieveWadhwa**
Added a step-by-step guide to whitelisting the database IP on AWS, resolving https://github.com/appsmithorg/appsmith/issues/935

### CODE REVIEW

File: core-concepts/connecting-to-databases/README.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ description: >-
 
 ## Database Connections
 
-Connections to databases can be created in the datasource pane. Contact your database admin for the connection credentials and configure them in the data source section. If you are hosting appsmith, you must whitelist the ip address of the appsmith deployment
+Connections to databases can be created in the datasource pane. Contact your database admin for the connection credentials and configure them in the data source section. If you are hosting appsmith, you must whitelist the IP address of the appsmith deployment
 
 {% hint style="success" %}
-Whitelist ****[18.223.74.85](https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#ElasticIpDetails:PublicIp=18.223.74.85) to connect to your database from our cloud hosted version
+Whitelist ****[18.223.74.85](https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#ElasticIpDetails:PublicIp=18.223.74.85) to connect to your database from our cloud hosted version. Step-by-step guide available [here](aws-whitelist.md)
```
Comment (mohanarpit) [inline core-concepts/connecting-to-databases/README.md:14]: Can we remove the URL link here? It doesn't link to anything useful.
```suggestion
Whitelist 18.223.74.85 to connect to your database from our cloud hosted version. A step-by-step guide to do this on AWS is available [here](aws-whitelist.md)
```

**Comment by mohanarpit** [comment]
@AtieveWadhwa I think there's been a miscommunication. The linked issue talks about whitelisting Appsmith's public IP on AWS RDS instances. Since databases are typically not exposed to the internet, the hosted version of Appsmith cannot connect to user's DB to execute a query. That's why we ask them to whitelist our IP on their RDS instances. 

We'd like to give steps to the user to whitelist Appsmith's public IP on their private RDS instance. Does this make sense?

**Comment by AtieveWadhwa** [comment]
@mohanarpit Sorry for misunderstanding! I'll make it again and commit the changes soon. Thanks for your help.

**Comment by mohanarpit** [comment]
@AtieveWadhwa Are you still working on this issue?

**Comment by AtieveWadhwa** [comment]
@mohanarpit Yes, I am working on this. I'm tied up with exams at the moment but I'll surely be pushing a new commit in a couple of days. Sorry for the delay!

**Comment by mohanarpit** [comment]
@AtieveWadhwa No worries. I'm unassigning you from this issue for the time being. Please ask to be assigned to this issue whenever you have time. 

All the best for your exams! 😄

**Pull request closed without merging**